Makefile

```diff
@@ -1,7 +1,7 @@
 NAME		=	webserv
 INCLUDES	=	-I include
 
```

Comment:
yeah i think leaving  while we work, but i think we should always put it back on your final push for the PR